packages/lesswrong/server/inactiveUserSurveyCron.tsx

```diff
@@ -29,7 +29,7 @@ const sendInactiveUserSurveyEmails = async () => {
     try {
       void wrapAndSendEmail({
         user,
-        from: 'eaforum@centreforeffectivealtruism.org',
+        from: 'EA Forum Team <eaforum@centreforeffectivealtruism.org>',
```

Comment:
I sign this email with my name (`- Sarah (for the EA Forum Team)`) so maybe I should do that here, but this also seems fine